### ISSUE TITLE
List component accessibility violations

### ISSUE THREAD

**Issue by luketa8** [grommet/hpe-design-system#4366]
### What does this request relate to?

Grommet

### Describe the feature you'd like to request.

- In [Filter List](https://unrivaled-bublanina-3a9bae.netlify.app/?path=/story/filtering--filter-lists), ARIA input fields must have an accessible name. 

Currently:

> aria-label attribute does not exist or is empty
> aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
> Element has no title attribute

- Add `aria-label` attribute to Lists
- Add `aria-labelledby` attribute to Lists
- Add `title` attribute to Lists

### How urgent is this request?

None
